### DISCUSSION TITLE
2023/05/20/lithopedia-part-1-intro-and-clay-experiments

### DISCUSSION THREAD

**Discussion by giscus** [pepijndevos/pepijndevos.github.com#3] [Announcements]
# 2023/05/20/lithopedia-part-1-intro-and-clay-experiments

In this digital age so much of our information is only on the internet, and it’s hard to predict how much of it would survive another Carrington event. So what if we engrave all of the information on the internet in stone? We have clay tablets dating back thousands of years, so if we want to preserve our knowledge for the millenia to come, it’s a time-proven method.

http://127.0.0.1:4000/2023/05/20/lithopedia-part-1-intro-and-clay-experiments.html

**Comment by pepijndevos** [comment]
Stay tuned for part two
